docs/qppe-server.yaml

```diff
@@ -886,7 +885,8 @@ components:
               type: number
               nullable: true
               format: double
-              description: A fraction between question.score_min_fraction and question.score_max_fraction.
+              description: The score for this question attempt, must lie between the `score_min` and `score_max` set by
+                the question.
```

Comment:
"must lie" klingt für mich nicht richtig (kann es aber auch nicht abschließend beurteilen). DeepL Write schlägt "must be" vor. Betrifft dann auch die Zeile unten.

Reply:
Da bin ich mir ziemlich sicher, dass es richtig ist, wird relativ viel benutzt: https://glosbe.com/en/en/lie%20between